cmd/vic-machine/validate/validator.go

```diff
@@ -350,6 +354,62 @@ func (v *Validator) network(ctx context.Context, input *data.Data, conf *metadat
 	}
 }
 
+func (v *Validator) GetVCH(input *data.Data) (*object.ResourcePool, error) {
+	rp, err := v.GetResourcePool(input)
+	if err != nil {
+		return nil, err
+	}
+	vchPath := fmt.Sprintf("%s/%s", rp.InventoryPath, input.DisplayName)
+	vchPool, err := v.Session.Finder.ResourcePool(v.Context, vchPath)
+	if err != nil {
+		log.Errorf("Failed to get VCH resource pool %s: %s", vchPath, err)
+		return nil, err
+	}
+	return vchPool, nil
+}
+
+func (v *Validator) GetVCHConfig(vch *object.ResourcePool, name string) (*metadata.VirtualContainerHostConfigSpec, error) {
```

Comment:
This definitely doesn't belong in Validator - I think this would be part of the New... methods in the prior comment.